gcc/testsuite/gdc.test/runnable/test36.d

```diff
@@ -59,7 +63,7 @@ extern (Windows):
 int main()
 {
         auto inst= new invarianttest;
-        int esp;
+        size_t esp;
```

Comment:
I think that this breaks x86

Reply:
Hmm, OK. Is it actually intended that this test saves only 32 bit of the 64bit sp on 64 bit architectures? I guess I'll just change that back to `int` and change the AArch64 asm to save only 32 bit, as is done for X86?

Reply:
```
test36.d: Assembler messages:
test36.d:39: Error: unsupported instruction `mov'
test36.d:71: Error: unsupported instruction `mov'
test36.d:94: Error: unsupported instruction `mov'
```
Just use `%%rsp` for the 64bit path.

Reply:
There's also X32 to consider, which will complain as well, but that's been untested for years, and probably has more problems to fix.

Reply:
I just save 32 bits for AArch64 now (like X86_32), leaving other targets unaffected.